src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -193,11 +194,17 @@ def main(
     else:
         logger.info(f"Found {len(job_parameters)} jobs to run.")
 
+    num_workers = (
+        extra_args["max_workers"]
+        and min(extra_args["max_workers"], len(job_parameters))
+        or len(job_parameters)
+    )
+
     logger.info("Spinning up Redis DBs and connecting to main process.")
```

Comment:
Can you explain the logic here?

Reply:
if max_workers is truthy, return min (max_workers, num_jobs)
else return num_jobs
this had the kind of nice property of evaluating to num_jobs if 0 was passed into max_workers, but in the end I just restricted that through the click option
On reflection, it's probably not best to use the idiomatic and/or ternary construction

Reply:
Nvm, I see you changed the expression completely.

Reply:
Could alternately do this:
```python
num_workers = len(job_parameters)
if extra_args["max_workers"]:
    num_workers = min(extra_args["max_workers"], num_workers)
```